PowerShellGet/PSModule.psm1

```diff
@@ -13803,6 +13887,61 @@ function Get-PrivateData
     }
 }
 
+function Validate-PreReleaseString 
+{
+    [CmdletBinding(PositionalBinding=$false)]

```

Comment:
CmdletBinding will add cmdlet semantics like -Verbose,-Debug,-Warning common parameters and generally may require additional processing. So removing that may make your code faster.